Widescreen patch
Hoax - Cutscene: Aftermath
Knight: prevent hiding after camera pan, rotate back to face fallen knight
--- original
+++ modified
@@ -167,5 +167,9 @@
 yield
 call 257, stor[24], stor[25]
 jmp_cmp <, 0xffffffe7, stor[24], inl[:LABEL_24]
-call 106, 0x1
+mov 0x6, stor[24]
+mov 0x9, stor[25]
+mov 0x0, stor[26]
+gosub inl[:LABEL_8]
+call 99, 1
 return
